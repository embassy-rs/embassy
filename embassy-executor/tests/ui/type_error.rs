#![cfg_attr(feature = "nightly", feature(impl_trait_in_assoc_type))]

#[embassy_executor::task]
async fn task() {
    5
}

fn main() {}
