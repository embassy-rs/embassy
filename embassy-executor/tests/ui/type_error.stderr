error[E0308]: mismatched types
 --> tests/ui/type_error.rs:5:5
  |
4 | async fn task() {
  |                - help: try adding a return type: `-> i32`
5 |     5
  |     ^ expected `()`, found integer
